WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Shipping Address Validation/ShippingLabelAddressValidationViewController.swift

```diff
@@ -0,0 +1,309 @@
+import UIKit
+import Yosemite
+
+final class ShippingLabelAddressValidationViewController: UIViewController {
```

Comment:
nit: validation seems to be an implementation detail within the address form, how about `ShippingLabelAddressFormViewController`?